src/php/Lang/Collections/Vector/TransientVectorInterface.php

```diff
@@ -6,11 +6,13 @@
 
 use ArrayAccess;
 use Countable;
+use IteratorAggregate;
 
 /**
  * @template T
+ * @extends IteratorAggregate
```

Comment:
Why only `IteratorAggregate`?

Reply:
Actually, I think this line is not necessary at all here...